src/main/webapp/app/pages/HomePage.tsx

```diff
@@ -46,6 +46,7 @@ interface IHomeProps {
 export type ExtendedTypeaheadSearchResp = TypeaheadSearchResp & {
   alterationsName: string;
   tumorTypesName: string;
+  cancerName: string;
```

Comment:
Can we reuse tumorTypesName?

Reply:
that's for a list of cancer that a drug search returns, do you think we should just keep one list of cancer and determine based on if it came from drug or cancer_type search?